src/OpenConext/ProfileBundle/Resources/views/MyConnections/overview.html.twig

```diff
@@ -6,28 +6,24 @@
 
 {% block content %}
 
-    {% if orcid is null %}
-        <h1>{{ 'profile.my_connections.short_title'|trans }}</h1>
-    {% elseif orcid.isConnected %}
-        <h1>{{ 'profile.my_connections.short_title'|trans }}</h1>
-    {% else %}
-        <h1>{{ 'profile.my_connections.long_title'|trans }}</h1>
-        <p>{{ 'profile.my_connections.explanation'|trans }}</p>
-    {% endif %}
+    <h1>{{ 'profile.my_connections.long_title'|trans }}</h1>
+    <p>{{ 'profile.my_connections.explanation'|trans }}</p>
 
     <div class="table-overflow-container">
-        {% if orcid is null %}
-            {{ 'profile.my_connections.no_connections_configured.description'|trans }}
-        {% elseif orcid.isConnected %}
+    <h2>{{ 'profile.my_connections.active_connections'|trans }}</h2>
+    {% if activeConnections is empty %}
+        {{ 'profile.my_connections.no_active_connections.description'|trans }}
+    {% else %}
+        {% for connection in activeConnections %}
             <div class="attribute-container">
                 <div class="mdl-grid">
                     <div class="mdl-cell mdl-cell--3-col">
                         <div class="mdl-layout-title">{{ 'profile.my_connections.service_label'|trans }}</div>
-                        <img src="{{ orcid.logoPath }}" title="{{ 'profile.my_connections.orcid.title'|trans }}"/>
+                        <img src="{{ connection.logoPath }}" title="{{ 'profile.my_connections.orcid.title'|trans }}"/>
```

Comment:
If this template now handles connections in a generic manner, shouldn't we remove all references to orcid in placeholders?

Reply:
I propose to do this when the next connection is introduced. The generic solution was primarily introduced to be able to display headings and texts without too much hassle.